src/OrgV2Factory.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./OrgV1.sol";
+
+interface SafeFactory {
+    function createProxy(address masterCopy, bytes memory data) external returns (Safe);
+}
+
+interface Resolver {
+    function multicall(bytes[] calldata data) external returns(bytes[] memory results);
+    function setAddr(bytes32, address) external;
```

Comment:
Minor semantic change; do we want to add the names to the interface methods?

```solidity
function setAddr(bytes32 node, address addr) external;
```

https://docs.ens.domains/contract-api-reference/publicresolver#set-ethereum-address


I don't think it affects the method signature, since only parameter types are used in the signature.

Reply:
Yeah I was getting annoying warnings for some reason when I had the parameter names, due to a method with the same name existing... eg. `addr()` and `address addr`. That's why I removed them in some places, but not sure why the warning exists.

Reply:
Confirmed, annoying warnings! That's too bad.. not sure why that is as well. Possible workaround is prefixing all interface parameter names with an underscore. I've tried this just now and it seems to do the trick. Not sure if that is a solution for the underlying problem, though.

Something like this is workable:

```solidity
interface SafeFactory {
    function createProxy(address _masterCopy, bytes memory _data) external returns (Safe);
}

interface Resolver {
    function multicall(bytes[] calldata _data) external returns(bytes[] memory results);
    function setAddr(bytes32 _node, address _addr) external;
    function addr(bytes32 _node) external returns (address);
    function name(bytes32 _node) external returns (string memory);
}

interface Registrar {
    function commit(bytes32 _commitment) external;
    function register(string calldata _name, address _owner, uint256 _salt) external;
    function ens() external view returns (address);
    function radNode() external view returns (bytes32);
    function registrationFeeRad() external view returns (uint256);
    function minCommitmentAge() external view returns (uint256);
}

interface Safe {
    function setup(
        address[] calldata _owners,
        uint256 _threshold,
        address _to,
        bytes calldata _data,
        address _fallbackHandler,
        address _paymentToken,
        uint256 _payment,
        address payable _paymentReceiver
    ) external;

    function getThreshold() external returns (uint256);
    function isOwner(address _owner) external returns (bool);
}
```

Reply:
Yeah indeed, but it doesn't really matter, the interface is for internal use only and the parameter name only serves as documentation.